packages/web/tt.js

```diff
@@ -0,0 +1,543 @@
+      <h1>Help Translate Journaly!</h1>
+    </header>
+    <p>
+      Journaly is currently accepting translations in the ${LOCALES.length}
+      languages other than English. We'd love to have your help improving these
+      or translating the UI into new languages!
+    </p>
+    <p>
+      To help translate one of the existing languages, find it in the table
+      below, download the linked template, fill it out and email it to
+      <a href="mailto:hello@journaly.com">hello@journally.com</a>.
+    </p>
+    <p>
+      To translate a language that isn't listed, drop us a line at
+      <a href="mailto:hello@journaly.com">hello@journally.com</a>.
+      and we'll look at adding it. Since not every language has the same usage
+      among our userbase, and because each translation requires maintainance
+      effort, we may hold on adding some languages until we find there is a
+      significant demand, but generally we're open to translations. Contact
+      us if you're not sure!
+    </p>
+
+    <h2>Filling out a Translation Template</h2>
+    <p>
+      To fill out a template, simply look at the "Source String" column which
+      contains the original English text, and enter the translation in the
+      "Translated String" column. Some strings contain variables which are
+      represented as "{{variable_name}}", which will be substituted with a
+      value on the site. For example the read time string is
+      "{{minutes}} min. read" in English and  "{{minutes}} Min. Lesezeit" in
+      German. Use the same variable name in your translation, move it around if
+      that's appropriate for the target language (e.g. it does not need to be
+      at the start of the string in the target langauge, just because it's at
+      the start in English), but keep the name the same.
+    </p>
+    <p>
+      The "Translator Notes" column is not processed by the system, but it's
+      there as a way for you, the translator, to communicate with us as
+      developers. Feel free to ask questions, explain a particular translation,
+      or generally record your thoughts. Your TL notes won't be revealed to
+      anyone besides the core Journaly team.
+    </p>
+    <p>
+      Please do not edit any columns other than "Translated String" and
+      "Translator Notes" as this affects our ability to automatically process
+      your submission.
+    </p>
+    <p>
+      The "status" column records what the system has marked the translation as,
+      e.g. is the translation missing, or has the English copy changed since the
+      last time a translation was submitted. Sometimes translations will be
+      marked out of date, even if they're still valid. If this is the case,
+      simply add a translator note indicating it, and we'll mark it as up to
+      date.
+    </p>
+    <p>
+      Also feel free to review translations that are marked as being up to date.
+      We assume good faith and do not independently verify every translation,
```

Comment:
```suggestion
      We generally trust the submissions we get and do not independently verify every translation,
```

(this is another small suggestion just to make it easier to understand for non-natives)